src/components/pastAnalysis/PastAnalysisTable.tsx

```diff
@@ -47,7 +48,7 @@ const PastAnalysisTable = () => {
       title: 'Study',
       cellFormatters: [expandable]
     },
-    'Patient MRN', 'Patient DOB', 'Patient Age', 'Analysis Created'
+    'Patient MRN', 'Patient DOB', 'Patient Age', 'Analysis Created', ''
```

Comment:
@janakitti can you do me a favor and replace single quotes with double quotes? Thanks